test/PoolSwapLibrary/getPrice.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { ethers } from "hardhat"
+import chai from "chai"
+import chaiAsPromised from "chai-as-promised"
+import { PoolSwapLibrary, PoolSwapLibrary__factory } from "../../types"
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+
+chai.use(chaiAsPromised)
+const { expect } = chai
+
+describe("PoolSwapLibrary - getPrice", () => {
+    let signers: SignerWithAddress[]
+    let library: PoolSwapLibrary
+    beforeEach(async () => {
+        // Deploy the contracts
+        signers = await ethers.getSigners()
+
+        const libraryFactory = (await ethers.getContractFactory(
+            "PoolSwapLibrary",
+            signers[0]
+        )) as PoolSwapLibrary__factory
+
+        library = await libraryFactory.deploy()
+        await library.deployed()
+    })
+
+    describe("When sideBalance is 0", () => {
+        it("returns 0", async () => {
+            const sideBalance = ethers.utils.parseEther("0")
+            const tokenSupply = ethers.utils.parseEther("101230")
+
+            const price = await library.getPrice(sideBalance, tokenSupply)
+
+            const zero = "0x00000000000000000000000000000000".toLowerCase() // 0 in IEE754 binary128
+            expect(price.toLowerCase()).to.equal(zero)
+        })
+    })
+
+    describe("When totalSupply is 0", () => {
+        it("returns 1", async () => {
+            const sideBalance = ethers.utils.parseEther("10000")
+            const tokenSupply = ethers.utils.parseEther("0")
+
+            const price = await library.getPrice(sideBalance, tokenSupply)
+
+            const one = "0x3fff0000000000000000000000000000".toLowerCase() // 1 in IEE754 binary128
```

Comment:
I don't think it would be -- we'd just need to create a mock contract that exposes this function externally: https://github.com/abdk-consulting/abdk-libraries-solidity/blob/master/ABDKMathQuad.sol#L46-L63

But yeah, understood it can be done another time; your call if you think the above is worth doing in this PR

Reply:
Let's just get this one merged in and do it in a separate PR